src/code/z_player_lib.c

```diff
@@ -51,26 +53,23 @@ Vec3f D_801F59B0[2];
 
 s32 D_801F59C8[2];
 
-typedef struct {
-    /* 0x0 */ s16 unk_0;
-    /* 0x2 */ s16 unk_2;
-    /* 0x4 */ s16 unk_4;
-    /* 0x6 */ s16 unk_6;
-    /* 0x8 */ s16 unk_8;
-} struct_801F59D0; // size = 0xA
+typedef struct BunnyEarKinematics {
+    /* 0x0 */ Vec3s rot;
+    /* 0x6 */ Vec3s angVel;
+} BunnyEarKinematics; // size = 0xC
 
-struct_801F59D0 D_801F59D0;
+BunnyEarKinematics sBunnyEarKinematics;
 
-Vec3f* D_801F59DC;
+Vec3f* sPlayerCurBodyPartPos;
 
 s32 D_801F59E0;
 
-s32 D_801F59E4;
+s32 sPlayerLod;
```

Comment:
I suppose this could be an enum near/far (or high/low since it's a *level* of detail), I don't see it being treated as a boolean anywhere. It's used as an array access in the limbs, I think? I'd have thought it was only worth doing if it can be done in the arrows too (and anything else that uses lod limbs).